{"name": "test_get_user_orders_authorized", "status": "broken", "statusDetails": {"message": "_pytest.fixtures.FixtureLookupError: ('client', <FixtureRequest for <Function test_get_user_orders_authorized>>)", "trace": "file C:\\Python\\qa_python_projects\\Diplom\\Diplom_2\\tests\\test_order_retrieval.py, line 1\n  def test_get_user_orders_authorized(client):\nE       fixture 'client' not found\n>       available fixtures: cache, capfd, capfdbinary, caplog, capsys, capsysbinary, cov, doctest_namespace, monkeypatch, no_cover, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory\n>       use 'pytest --fixtures [testpath]' for help on them.\n\nC:\\Python\\qa_python_projects\\Diplom\\Diplom_2\\tests\\test_order_retrieval.py:1"}, "start": 1730209339768, "stop": 1730209339768, "uuid": "312b2a29-89aa-4438-9522-21a2ada4bece", "historyId": "1a4b42c54aa91c7b47a4fcaa34a0e360", "testCaseId": "1a4b42c54aa91c7b47a4fcaa34a0e360", "fullName": "tests.test_order_retrieval#test_get_user_orders_authorized", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_order_retrieval"}, {"name": "host", "value": "WorkPC"}, {"name": "thread", "value": "14480-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_order_retrieval"}]}